test/forge/FeeTest.sol

```diff
@@ -40,6 +40,17 @@ contract FeeTest is IntegrationTest {
         _setCap(allMarkets[0], CAP);
     }
 
+    function testSetFee(uint256 fee) public {
+        fee = bound(fee, 0, ConstantsLib.MAX_FEE);
+
+        vm.expectEmit();
```

Comment:
```suggestion
        vm.expectEmit(address(vault));
```

Reply:
I feel that it should be part of a PR on top instead since there are much more occurences